calico-cloud_versioned_docs/version-3.17/release-notes/index.mdx

```diff
@@ -4,6 +4,44 @@ description: What's new, and why features provide value for upgrading.
 
 # Release notes
 
+## Calico Cloud v17.0.0
+
+August 18, 2023
+
+v17.0 is the first versioned release of {{prodname}}.
```

Comment:
Let's remove this line as we are tracking versions internally

Reply:
Removed.